src/libsystemd-network/dhcp-static-lease.c

```diff
@@ -0,0 +1,60 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <errno.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "alloc-util.h"
+#include "dhcp-internal.h"
+#include "dhcp-server-internal.h"
+#include "memory-util.h"
+#include "strv.h"
+#include "utf8.h"
+
+static sd_dhcp_static_lease* sd_dhcp_static_lease_free(sd_dhcp_static_lease *i) {
```

Comment:
usually our destructors have a `if (!i) return NULL;` line as first step, so that it is safe to call them wiht a NULL argument. This one should hvae that too I think, out of a matter of principle